Building the unit tests fails with GNAT 11.2 and AUnit 21.0 provided by Alire:

```
UNITTEST tests/unit/
aunit.gpr:18:24: library directory "/home/runner/work/ada-runtime/ada-runtime/build/gnat_11/alire/cache/dependencies/aunit_21.0.0_d6561681/lib/aunit/native-full/" does not exist
aunit.gpr:22:23: object directory "../aunit-obj/native-full" not found
gprbuild: "test" processing failed
make: *** [Makefile:70: tests/unit/test] Error 4
```

@jklmnn Can you take a look at this please?

--- a/Makefile
+++ b/Makefile
@@ -59,15 +59,15 @@ TEST_BINS = $(addsuffix /test,$(TEST_DIRS))
 
 $(TEST_DIR)/%/test:
 	@echo "TEST $(dir $@)"
-	$(VERBOSE)cd $(dir $@) && gprbuild -q --RTS=../../../build/posix/obj -p && ./test
+	$(VERBOSE)cd $(dir $@) && gprbuild -p -q --RTS=../../../build/posix/obj -p && ./test
 
 $(TEST_DIR)/exception/test:
 	@echo "TEST $(dir $@)"
-	$(VERBOSE)cd $(dir $@) && gprbuild -q --RTS=../../../build/posix/obj -p && ./test; test $$? -gt 0
+	$(VERBOSE)cd $(dir $@) && gprbuild -p -q --RTS=../../../build/posix/obj -p && ./test; test $$? -gt 0
 
 $(UNIT_DIR)/test:
 	@echo "UNITTEST $(dir $@)"
-	$(VERBOSE)cd $(dir $@) && gprbuild -q -P test && ./test
+	$(VERBOSE)cd $(dir $@) && gprbuild -p -q -P test && ./test
 
 test: posix clean_test $(TEST_BINS) $(UNIT_DIR)/test
 

